### DISCUSSION TITLE
Copy/Paste Generation Data

### DISCUSSION THREAD

**Discussion by ed1g1tal** [vladmandic/sdnext#2893] [General]
I thought at one point (perhaps just on A1111) there was a way to copy image generation data (from Civitai, for example) into a pastable area in Text to Image. Is this an extension or setting that I am missing or just not available in SD.Next? Thank you.

**Comment by vladmandic** [comment]
It's still there.
You do need to press "restore" button so it gets parsed and settings restored from that prompt.

**Comment by ed1g1tal** [comment]
Hi, Vlad -

  Thank you for the quick reply. 

  I don't think Restore is what I was referring to. That seems to restore the last prompt that I entered.
  
  I am talking about where you can click Copy Generation Data on Civitai like below. That copies to clipboard. 
  
  
![image](https://github.com/vladmandic/automatic/assets/107633253/79997bfc-2005-449c-b97c-efcb8f751c7c)

Is there a main feature or extension you are aware of that would read the clipboard data to then populate it to Text2Image in SD.Next?

**Comment by vladmandic** [comment]
paste generation data to prompt and when its there, click restore. that's it.

**Comment by ed1g1tal** [comment]
That worked. Thank you. :D
